specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2022-08-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,8364 @@
+        },
+        "limit": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of suggestions we hope to return. The default value is 5. The value must be a number between 1 and 100."
+        },
+        "filter": {
+          "type": "object",
+          "description": "The filter for the search."
+        }
+      },
+      "title": "SuggestRequest",
+      "description": "The query of suggest request."
+    },
+    "ImportCSVOperation": {
+      "type": "object",
+      "title": "Status of import csv operation",
+      "description": "The status of import csv operation",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "guid string"
+        },
+        "status": {
+          "$ref": "#/definitions/ImportCSVOperationStatus"
+        },
+        "createTime": {
+          "type": "string",
+          "description": "The created time of the record."
+        },
+        "lastUpdateTime": {
+          "type": "string",
+          "description": "The last updated time of the record."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ImportCSVOperationProperties"
+        },
+        "error": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AsyncOperationError"
+        }
+      }
+    },
+    "ImportCSVOperationProperties": {
+      "type": "object",
+      "properties": {
+        "importedTerms": {
+          "type": "string",
+          "description": "Term numbers that already imported successfully"
+        },
+        "totalTermsDetected": {
+          "type": "string",
+          "description": "Total term numbers that detected in csv"
+        }
+      }
+    },
+    "ImportCSVOperationStatus": {
+      "type": "string",
+      "enum": [
+        "NotStarted",
+        "Succeeded",
+        "Failed",
+        "Running"
+      ],
+      "x-ms-enum": {
+        "name": "ImportCSVOperationStatus",
+        "modelAsString": true
+      },
+      "title": "ImportCSVOperationStatus",
+      "description": "Enum of the status of import csv operation."
+    },
+    "BulkDeleteOperation": {
+      "type": "object",
+      "title": "Status of bulk delete operation",
+      "description": "The status of bulk delete operation",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "guid string"
+        },
+        "status": {
+          "$ref": "#/definitions/BulkDeleteOperationStatus"
+        },
+        "createTime": {
+          "type": "string",
+          "description": "The created time of the record."
+        },
+        "lastUpdateTime": {
+          "type": "string",
+          "description": "The last updated time of the record."
+        },
+        "properties": {
```

Comment:
I hear that is deployed.

But I remember recommendation is "result"?

Reply:
We only changed error schema in this round according to the recommendation. The rest remains unchanged.